catalyst_voices/packages/internal/catalyst_voices_models/lib/src/proposal_schema/proposal_schema.dart

```diff
@@ -0,0 +1,97 @@
+import 'package:catalyst_voices_models/src/proposal_schema/proposal_definitions.dart';
+
+class ProposalSchema {
+  final String schema;
+  final String title;
+  final String description;
+  final ProposalDefinitions definitions;
```

Comment:
i don't think "Scheme" needs `definitions`, its necessary for creating correct properties only.